chainntnfs/txnotifier.go

```diff
@@ -10,14 +10,28 @@ import (
 	"github.com/btcsuite/btcutil"
 )
 
+const (
+	// ReorgSafetyLimit is the chain depth beyond which it is assumed a
+	// block will not be reorganized out of the chain. This is used to
+	// determine when to prune old confirmation requests so that reorgs are
+	// handled correctly. The average number of blocks in a day is a
+	// reasonable value to use.
+	ReorgSafetyLimit = 144
+
+	// MaxNumConfs is the maximum number of confirmations that can be
+	// requested on a transaction.
+	MaxNumConfs = ReorgSafetyLimit
```

Comment:
Any reason why we won't set this to a different number? 144 seems a bit high for `MaxConfs` IMO.

Reply:
Not really, it just needs to be `<= ReorgSafetyLimit` to ensure notifications are properly tracked within the `TxNotifier`.

Reply:
Should we set it lower?

Reply:
Sure, but we must also keep in mind that lowering this also impacts the number of confirmations clients can request to the `ChainNotifier`. Do you have any suggestions on a lower number?

Reply:
143? Nah, can probably keep this as is 😛